### DISCUSSION TITLE
How to use `nicegui-pack` with `--upx`??

### DISCUSSION THREAD

**Discussion by KiddoV** [zauberzeug/nicegui#3786] [Q&A]
### Question

How do I package the app with `upx` to make the `exe` file smaller?

Thanks,

**Comment by python-and-novella** [comment]
You can package it with pyintaller which supports.

**Reply by KiddoV** [reply]
Would be nice that `nicegui-pack` has all the necessary `CLI arguments` too. Specially, for this.
